packages/@aws-cdk/aws-apigateway/package.json

```diff
@@ -285,7 +285,8 @@
       "props-default-doc:@aws-cdk/aws-apigateway.MethodOptions.operationName",
       "props-default-doc:@aws-cdk/aws-apigateway.MethodOptions.requestModels",
       "props-default-doc:@aws-cdk/aws-apigateway.MethodOptions.requestValidator",
-      "docs-public-apis:@aws-cdk/aws-apigateway.ResourceBase.url"
+      "docs-public-apis:@aws-cdk/aws-apigateway.ResourceBase.url",
+      "attribute-tag:@aws-cdk/aws-apigateway.TokenAuthorizer.authorizerArn"
```

Comment:
why exclude? Just add an @attribute tag, no?

Reply:
CloudFormation does not return the ARN - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-authorizer.html#aws-resource-apigateway-authorizer-return-values